onnx/backend/test/cpp/test_main.cc

```diff
@@ -2,11 +2,12 @@
 #include "gtest_utils.h"
 
 GTEST_API_ int main(int argc, char** argv) {
-  if (argc != 2) {
+  if (argc < 2) {
     std::cerr << "target directory must be given!" << std::endl;
     return EXIT_FAILURE;
   }
-  std::string target_dir = argv[1];
+  argc--;
```

Comment:
later, argc is used again by init google test, do we want expose the change on argc to initgoogletest?

Reply:
@houseroad Yes, because I dont want google test to receive this dir anymore.